content: add placeholder content (#5)
open-education-hub/oer-template: content/chapters/template-chapter/lecture/slides/content.md

## Slides hierarchy

> ----

Reviewer: ```suggestion
---
```

> ## Slides hierarchy

Reviewer: ```suggestion
## Slides Hierarchy
```

* Slides are organized in two levels

>     * We use `---` in order to mark a new idea per slides; this leads to new horizontal slides

Reviewer: ```suggestion
    * We use `---` in order to open up a new topic.
       This leads to a new horizontal slide figuring a topic that can be expanded by vertical slides.
```

>     * When talking about the same subject, but we want to use a new slide, we use `----`, which adds a new vertical slide

Reviewer: ```suggestion
    * We expand discussions on the same topic by vertical slides.
       We use `----`, which adds a new vertical slide
```

----

> ### New vertical slide

Reviewer: ```suggestion
### New Vertical Slide
```

* This slide ilustrates a new item of discusion on the matter of the slides hierarchy

---

> ### Animated slides

Reviewer: ```suggestion
### Animated Slides
```

* You can even animate slides like this

``` [1 - 2 | 3 - 4]
student@os:~/.../compute/lecture/demo/create-process$ strace -e clone ./fork_exec
clone(child_stack=NULL, flags=CLONE_CHILD_CLEARTID|CLONE_CHILD_SETTID|SIGCHLD, child_tidptr=0x7f7e83aa4810) = 5302
student@os:~/.../compute/lecture/demo/create-thread$ strace -e clone ./create_thread
clone(child_stack=0x7f9ea7df0fb0, flags=CLONE_VM|CLONE_FS|CLONE_FILES|CLONE_SIGHAND|CLONE_THREAD|CLONE_SYSVSEM|CLONE_SETTLS|CLONE_PARENT_SETTID|CLONE_CHILD_CLEARTID, tls=0x7f9ea7df1700, child_tidptr=0x7f9ea7df19d0) = 5389
```

---

### Tables

> * Last, you can include tables

Reviewer: ```suggestion
You can include tables:
```
